tests/shared_test_functions.sh

```diff
@@ -181,15 +181,22 @@ ensure_valgrind_suppression() {
 	printf "done.\n"
 }
 
-## setup_check_variables ():
+## setup_check_variables (description):
 # Set up the "check" variables ${c_exitfile} and ${c_valgrind_cmd}, the
 # latter depending on the previously-defined ${c_valgrind_min}.
 # Advances the number of checks ${s_count} so that the next call to this
-# function will set up new filenames.
+# function will set up new filenames.  Write ${description} into a
+# file.
```

Comment:
I feel like there might be an intended "and print it if a test fails" or something like that?

Reply:
Technically, the "print it" doesn't happen in the `setup_check_variables` function.  But yes, I should add such a comment to the description of `notify_success_or_fail`.